mitodl/mitxpro
# Registration detail form - Step 1

#### Pre-Flight checklist



#### What are the relevant tickets?
Partially addresses #190 (Step 1 of the registration form)

#### What's this PR do?
renders Step 1 of the registration form (legal address fields) and saves a `LegalAddress` for new users.

#### How should this be manually tested?
- Register as a new user.
- Try out different values for country: state and zipcode fields should be available for US and Canada only
- Try adding multiple lines for street address
- Try submitting the form with and without all required fields
- After a successful submission, verify via django admin that your user has the correct `LegalAddress`


#### Screenshots (if appropriate)
![Screen Shot 2019-05-06 at 11 22 27 AM](https://user-images.githubusercontent.com/187676/57236361-e4a07300-6ff2-11e9-9abe-68c91233dde0.png)

![Screen Shot 2019-05-06 at 11 35 49 AM](https://user-images.githubusercontent.com/187676/57236473-29c4a500-6ff3-11e9-9308-03038f7ef9bb.png)


#### What GIF best describes this PR or how it makes you feel?
(Optional)
